Lib/test/test_inspect.py

```diff
@@ -1077,6 +1077,51 @@ class C(metaclass=M):
         attrs = [a[0] for a in inspect.getmembers(C)]
         self.assertNotIn('missing', attrs)
 
+class TestIsDataDescriptor(unittest.TestCase):
+
+    def test_custom_descriptors(self):
+        class NonDataDescriptor(object):
```

Comment:
Why do you specify the `object` class for the parent ? it is not required https://docs.python.org/3/reference/compound_stmts.html#class-definitions

Reply:
Keep the Python 3 code clear. Add the `object` class in a backport to Python 2.